packages/next/build/swc/index.js

```diff
@@ -34,6 +34,7 @@ for (const triple of triples) {
       loadError = e
       break
```

Comment:
I think it's actually this `break` that needs to be removed. I was thinking that if the file existed and it failed to load, it must be a problem with the file, but it seems like if a file fails to load that's a signal to check if there is another file that works.